themes/stanford_basic/stanford_basic.theme

```diff
@@ -642,3 +642,13 @@ function stanford_basic_preprocess_paragraph__stanford_spacer(&$variables) {
       ->getString();
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function stanford_basic_hook_preprocess_page__front( &$variables) {
+  /** @var \Drupal\config_pages\ConfigPagesLoaderServiceInterface $config_pages */
+
+    $config_pages = \Drupal::service('config_pages.site-information');
```

Comment:
hi @pookmish I need some feedback.  Is this the right way to get information from the config? Also there are two places where an author can set a site name. Which one would be the one to use? There is Basic Site Settings (/admin/config/system/site-information) and Site Settings (/admin/config/system/basic-site-settings). Thank you for your feedback.

Reply:
The easiest way is `$site_name = \Drupal::config('system.site)->get('name');`